### PR TITLE
Add a postsubmit to push kubetest2 CI binaries

### PR DESCRIPTION
**Pull request by amwat**
xref: https://github.com/kubernetes/enhancements/issues/2464
xref: https://github.com/kubernetes-sigs/kubetest2/pull/120

/cc @spiffxp @BenTheElder

### CODE REVIEW

File: config/jobs/image-pushing/k8s-staging-kubetest2.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+postsubmits:
+  kubernetes-sigs/kubetest2:
+  - name: post-kubetest2-push-binaries
+    cluster: k8s-infra-prow-build-trusted
+    annotations:
+      testgrid-dashboards: sig-testing-kubetest2
+      testgrid-alert-email: kubernetes-engprod+alerts@google.com
```
Comment (spiffxp) [inline config/jobs/image-pushing/k8s-staging-kubetest2.yaml:7]: sig-testing alerts/failure email should be here (instead?). Don't remember what it is off the top of my head

**Review by spiffxp — CHANGES_REQUESTED** [review]
One request

File: config/jobs/image-pushing/k8s-staging-kubetest2.yaml (hunk 1 above)
Comment (amwat) [reply config/jobs/image-pushing/k8s-staging-kubetest2.yaml:7]: kubernetes-sig-testing-alerts@googlegroups.com ?

Comment (amwat) [reply config/jobs/image-pushing/k8s-staging-kubetest2.yaml]: done

**Comment by amwat** [comment]
ref: https://github.com/kubernetes-sigs/kubetest2/issues/106

**Pull request merged**